Without the --encoding=utf-8 option i got these errors:

```
 [ERROR] [Package: terminal42/contao-leads; Language: de]: Property "description" does not pass the spell checker.      
         Either update the allow list or fix the spelling :) Errors: verf?, gbar, k?, nnen, zus?, tzlich, W?, nschen,   
         f?, Verf?, gung, f?, Unterst?, tzung  
```

Im not sure if user should add configuration to aspell, or we add this option to the linter.

https://github.com/contao/package-metadata/pull/723/changes/25759c5cf7cc4eaa47f5ce20ffa4b4805dab60c3#diff-19bb70feb6596d06459638b71ea12ed7e1a449b8f0c17a1d283d3823469c939bR31

--- a/meta/terminal42/contao-leads/de.yml
+++ b/meta/terminal42/contao-leads/de.yml
@@ -1,7 +1,7 @@
 de:
     title: Leads
     description: >
-        Mit dieser Erweiterung kannst du Formulardaten in Leads speichern und verwalten. Jeder Lead besteht aus einem Hauptformular und optionalen Unterformularen. Das Hauptformular definiert, welche Felder verfügbar sind. Dieser Ansatz hilft dir z.B. mehrsprachige Formulare zu erstellen und alle Daten in demselben Lead zu speichern.
+        Mit dieser Erweiterung kannst du Formulardaten in Leads speichern und verwalten. Jeder Lead besteht aus einem Hauptformular und optionalen Unterformularen. Das Hauptformular definiert, welche Felder verfügbar sind. Dieser Ansatz hilft dir z. B., mehrsprachige Formulare zu erstellen und alle Daten in demselben Lead zu speichern.
 
         Alle Konfigurationen können im Formulargenerator von Contao vorgenommen werden.
 

